test/System/Workflow/PerforatedScreenByImage/PanelWall.dyn

```diff
@@ -1,4 +1,4 @@
-<Workspace Version="0.8.1.1310" X="-329.867792183962" Y="-810.6891383816" zoom="0.653999019844256" Name="Home" RunType="Automatic" RunPeriod="100" HasRunWithoutCrash="True">
+<Workspace Version="0.8.1.1310" X="-329.867792183962" Y="-810.6891383816" zoom="0.653999019844256" Name="Home" RunType="Manual" RunPeriod="100" HasRunWithoutCrash="True">
```

Comment:
If we do not change the RunType, before we have a chance to change the path in the filename node, the graph already runs and fails.